packages/client/src/components/Modals/SearchPanel.vue

```diff
@@ -159,26 +158,43 @@ export default {
   },
   computed: {
     ...mapGetters({
-      fetchEligibilityCodes: 'grants/fetchEligibilityCodes',
+      searchFormFilters: 'grants/searchFormFilters',
+      eligibilityCodes: 'grants/eligibilityCodes',
     }),
   },
   methods: {
     ...mapActions({
       createSavedSearch: 'grants/createSavedSearch',
       updateSavedSearch: 'grants/updateSavedSearch',
+      applyFilters: 'grants/applyFilters',
+      fetchEligibilityCodes: 'grants/fetchEligibilityCodes',
     }),
     setup() {
-      // this.fetchEligibilityCodes();
+      this.fetchEligibilityCodes();
```

Comment:
I saw you'd commented out/removed this @as1729 , anything not working? `fetchEligibilityCodes` needed to be in actions, not getters, but otherwise worked as expected for me.

Reply:
I noticed errors in the console for this which is why I commented it out. Maybe now that it is in the actions it works well 👍